spec/index.bs

```diff
@@ -1761,6 +1801,80 @@ enum GPUCompareFunction {
 };
 </script>
 
+<dl dfn-type="enum-value" dfn-for=GPUCompareFunction>
+    : <dfn>"never"</dfn>
+    ::
+        Comparison tests never pass.
```

Comment:
what does it mean for the user that the comparison "passes", exactly?

Reply:
Addressed just above this in the explanation for a "comparison sampler". WDYT?